src/Microsoft.DotNet.Cli.Utils/MSBuildForwardingAppWithoutLogging.cs

```diff
@@ -23,9 +23,12 @@ internal class MSBuildForwardingAppWithoutLogging
             new Dictionary<string, string>
             {
                 { "MSBuildExtensionsPath", AppContext.BaseDirectory },
-                { "CscToolExe", GetRunCscPath() },
-                { "VbcToolExe", GetRunVbcPath() },
-                { "MSBuildSDKsPath", GetMSBuildSDKsPath() }
+                { "CscToolPath", GetRunToolPath() },
```

Comment:
`CscToolExe` is not supposed to be the full path to csc, but rather just the filename. `CscToolPath` is supposed to be the full containing directory of csc.